django/api/admin.py

```diff
@@ -46,7 +46,12 @@ def get_inlines(obj):
 
 @admin.register(GoElectricRebateApplication)
 class GoElectricRebateApplicationAdmin(admin.ModelAdmin):
-    exclude = ("sin",)
+    readonly_fields = (
+        "created",
+        "approved_on",
+        "not_approved_on",
+    )
+    exclude = ("sin", 'approved_on', 'not_approved_on', 'created')
```

Comment:
I'm not sure why "approved_on" and "not_approved_on" are under both "readonly_fields" and "exclude"; I think for this admin model, we can just leave it as it was with the one line "exclude = ("sin",)

Reply:
Thanks Tim, I just realized I left some lines in that needed to be removed. I'll make the necessary updates now.